### ISSUE TITLE
LOOT does not work with MO

### ISSUE THREAD

**Issue by Darcoon** [TanninOne/modorganizer#557]
When i try to use LOOT as an external program it gives me this error message:

Error code: -1; Failed to get game data. Details: boost::filesystem::file_size: The request is not supported: "C:\Program Files (x86)\Steam\steamapps\common\modname.esp"

This is strange because my mods nor my mod organizer is installed there. Loot is also not in there and fallout is in a separate map.  I tried both 64 bit and 32 bit loot both the same result.

when i try to use the build in loot i get kind of the same thing:
loot failed. Exit code was: 1

Using loot without MO works fine, i used it with NMM and just empty and it did fine.

Hope you will be able to fix it because i love MO and LOOT.

Windows 10 64 bit
FO4 mod organizer v2.0.4 (alpha 4)
 LOOT v0.8.1-53-gaac5583 (32 bit)
 LOOT v0.8.1-38-g7c4fe9f (64 bit)

**Comment by MysticDaedra** [comment]
I'm having very similar problems with LOOT. Sometimes it loads properly, sometimes it throws an error. It always crashes, and MO crashes with it, when I try to close LOOT to save the changes made. The built-in LOOT feature also isn't working for me, although this is probably a separate issue.

**Comment by TheGreenDragon** [comment]
I was able to get the latest LOOT dev build to function properly with MO by clicking Unlock in MO as LOOT loads. You have to be quick doing this, or I also get the errors you present. Once done LOOT functions properly. I tested this by moving an ESP above another ESP I know it should load after. LOOT sorted it properly. After applying the new load order and closing LOOT I switch from the Plugins tab to the Data tab and back to Plugins in MOA4 to see the changes.

**Comment by MysticDaedra** [comment]
So basically MO needs to be unlocked for LOOT to function? Thanks for the heads up. Wondering why this is though.

**Comment by Freso** [comment]
LOOT starts too fast for me to be able to unlock MO before LOOT has launched. If I create a LOOT shortcut via MO, close MO, and then launch LOOT via the MO created shortcut, LOOT seems to work for me. (`LOOT v0.8.1-236-gb347453 (32 bit).7z`, from the polymer-1.2 branch.)

**Comment by HashManRiver** [comment]
My LOOT Crashes and says invalid parameter [87]

**Comment by Freso** [comment]
Yeah. It's started showing the same message for me as what @HashManRiver describes now. :/

**Comment by TurboCartPig** [comment]
OK so this is pretty ghetto, but basically following these steps I have gotten it to work multiple times now.
1. Launch the Fallout 4 launcher via MO. (This messes up your load order and disables all plugins)
2. Run loot via MO. (Now and then this sorts stuff without even showing the UI, others you use it like intended)
3. Do not run loot again if the ui does not show, just check your load order and see if everything is ok. (running loot again gives all kinds of different errors.)
4. Come back here complaining it does not work. (Please don't)

I have no idea if this is just something on my end, but it does actually work for me. So I thought I would share.

**Comment by Gawdin** [comment]
I am having the same issue with it saying the failed to get game data error. This happens even if the only mod loaded is the official DLC (Automatron for example). I tried the shortcut trick, and while it doesn't give the error after doing that, it instead seems to be infinitely "Parsing, merging and evalutating metadata"
